cms/migrations/0003_product_detail_page.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 3.1.12 on 2021-07-29 09:13
+
+from django.db import migrations, models
+import wagtail.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cms", "0002_course_page"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="coursepage",
+            name="about",
+            field=wagtail.core.fields.RichTextField(
+                blank=True, help_text="About this course details.", null=True
+            ),
+        ),
+        migrations.AddField(
+            model_name="coursepage",
+            name="effort",
+            field=models.CharField(
+                blank=True,
+                help_text="A short description indicating how much effort is required (e.g. 1-3 hours per week).",
+                max_length=100,
+                null=True,
+            ),
+        ),
+        migrations.AddField(
+            model_name="coursepage",
+            name="length",
+            field=models.CharField(
+                blank=True,
+                help_text="A short description indicating how long it takes to complete (e.g. '4 weeks').",
+                max_length=50,
+                null=True,
+            ),
+        ),
+        migrations.AddField(
+            model_name="coursepage",
+            name="prerequisites",
+            field=models.CharField(
```

Comment:
Can we make this a rich text field? It seems like courses on edX have bullet lists and links in their prerequisites. (I can't remember if rich text fields support links).

from https://www.edx.org/course/microeconomics:

<img width="733" alt="image" src="https://user-images.githubusercontent.com/430126/127666285-fb30efde-b29f-4787-8050-4a349ac6ae38.png">

Reply:
I am going to add the price but I did not get what `Various` is pointing out and how to show the price if not having field in CMS  ?

Reply:
Field is richtext so can be used something like that. 


![screencapture-localhost-8013-homepage-data-analysis-for-social-scientist-2021-08-02-15_43_46](https://user-images.githubusercontent.com/7334669/127849618-37192c2f-7b60-4541-a9c9-58e4e9498a94.png)